### ISSUE TITLE
[Question] How do access locator with Page if not typed (TS)?

### ISSUE THREAD

**Issue by AveryTheDev** [microsoft/playwright#8323]
I want to use the locator method described on this [page](https://playwright.dev/docs/api/class-locator). However, I am getting a type error whenever I try to use the method with my fixture.

`Property 'locator' does not exist on type 'Page'.ts(2339)`

My fixture is setup like so

```ts
import { Page } from '@playwright/test';

type HelpPageFixture = {
  helpPage: Page;
};

const helpPageTest = test.extend<HelpPageFixture>({
  helpPage: async ({ page }, use) => {
  await page.goto('/help');
  
  await use(page)
  }  
});

helpPageTest('Navigate To Help Page', async ({ helpPage }) => { 
  const locator = helpPage.locator('text=Help'); //error, Property 'locator' does not exist on type 'Page'.ts(2339), here
});
```

I also tried importing Page from the `playwright` package itself and I still am getting the same error. I could not find Locator when I looked in the d.types.ts file either

My playwright version is:  `"^1.13.1"`
My @playwright/test version is: `"^1.12.3"`

**Comment by mxschmitt** [comment]
Locators were released with version 1.14, so upgrade `@playwright/test` to 1.14 should fix it. Also you don't need `playwright` when using `@playwright/test`.

**Comment by AveryTheDev** [comment]
Thank you! That did the trick
